.gitpod.yml

```diff
@@ -14,6 +14,9 @@ ports:
   - port: 9300
     onOpen: ignore
 tasks:
+  - before: >
```

Comment:
`nvm` is already available in the base image, so we don't need to do so much manual setup. Note the `--reinstall-packages-from=node` flag, which makes sure `yarn` will be available for the newly installed version too.

Since RVM is available too we should be able to manage Ruby the same way, but when I tried that I always ended up with 2.6.6 and I didn't want to invest more time in this.